src/TextArea/TextArea.less

```diff
@@ -0,0 +1,98 @@
+@import (reference) "../less/colors";
+@import (reference) "../less/border";
+@import (reference) "../less/sizing";
+@import (reference) "../less/type-size";
+@import (reference) "../less/type-utilities";
+@import (reference) "../less/spacing";
+
+.TextArea {
```

Comment:
Overall you should remove nesting except for explicit states. Most of these don't need to be nested since they are correctly using BEM.